documentation/src/main/asciidoc/topics/con_invalidation_caches.adoc

```diff
@@ -17,9 +25,8 @@ endif::downstream[]
 Sometimes the application reads a value from the external store and wants to write it to the local cache, without removing it from the other nodes.
 To do this, it must call `Cache.putForExternalRead(key, value)` instead of `Cache.put(key, value)`.
 
-Invalidation mode can be used with a shared cache store.
-A write operation will both update the shared store, and it would remove the stale values from the other nodes' memory.
-The benefit of this is twofold: network traffic is minimized as invalidation messages are very small compared to replicating the entire value, and also other caches in the cluster look up modified data in a lazy manner, only when needed.
+When you use invalidation mode with a shared cache store, write operations both update the shared store, and remove the stale values from the other nodes' memory.
```

Comment:
I can't imagine using invalidation cache with a non shared store... it only makes sense with a shared store as the nodes have to all be able to go the same source of truth to see the updated value.